docs/en/interfaces/formats.md

```diff
@@ -758,8 +758,7 @@ You can select data from a ClickHouse table and save them into some file in the
 clickhouse-client --query="SELECT * FROM {some_table} FORMAT Parquet" > {some_file.pq}
 ```
 
-Also look at the `HDFS` and `URL` storage engines to process data from the remote servers.
-
+To exchange data with the Hadoop ecosystem, you can use `HDFS` and `URL` table engines.
```

Comment:
* URL engine has nothing to do with Hadoop, so this sounds weird.
* "ecosystem" is too wide term, `HDFS` table engine and function work specifically with Hadoop and it's filesystem (which is called HDFS)
* HDFS+Parquet needs an example.
* "available to any project in the Hadoop ecosystem" above sounds weird, it's just one of a widespread formats in that ecosystem

Reply:
- URL engine allows to retrieve data from Hadoop through its HTTP interface?
- I'll do this example, when describing HDFS table engine.

Reply:
> URL engine allows to retrieve data from Hadoop through its HTTP interface?

Afaik it's not a common usage pattern for HDFS, available only via some extension. URL is more applicable to S3, but dedicated support for it is also coming soon in #5596